frontend/src/pages/create_campaign/index.tsx

```diff
@@ -30,10 +30,14 @@ const CreateCampaign = () => {
 
   const [tab, setTab] = useState(0);
   const [campaignName, setCampaignName] = useState("");
-  const [startDate, setStartDate] = useState(new Date());
-  const [endDate, setEndDate] = useState(
-    new Date(Date.now() + 14 * 3600 * 1000 * 24)
-  );
+  const [startDate, setStartDate] = useState(() => {
+    const initialState = new Date();
+    return initialState;
+  });
+  const [endDate, setEndDate] = useState(() => {
+    const initialState = new Date(Date.now() + 14 * 3600 * 1000 * 24);
+    return initialState;
+  });
```

Comment:
ah we should be able to just return directly here by having no braces, like `() => returnValue`. will prob need parentheses for the second one to put onto own line

sorry for all the comments lmao